src/controller/python/chip/clusters/Command.py

```diff
@@ -147,14 +149,17 @@ def SendCommand(future: Future, eventLoop, responseType: Type, device, commandPa
     '''
     if (responseType is not None) and (not issubclass(responseType, ClusterCommand)):
         raise ValueError("responseType must be a ClusterCommand or None")
+    if payload.must_use_timed_invoke and timedRequestTimeoutMs is None or timedRequestTimeoutMs == 0:
+        raise PermissionError(
```

Comment:
Odd error to emit - I'd think `ValueError` is a better fit, since it is indeed a case of invalid arguments?